src/factories/ModuleFactory_v1.sol

```diff
@@ -93,6 +94,8 @@ contract ModuleFactory_v1 is
     //--------------------------------------------------------------------------
     // Storage
 
+    address public immutable reverter;
```

Comment:
Currently, there is no real benefit of having it in this contract, as it's just stored. 

What we should do is to ensure that this `reverter` matches the one of the beacon whenever we `registerMetadata`, right? Otherwise, the user can never be sure we didn't just add a different one in the beacon without verifying themselves. If we check in `registerMetadata`, the user can be sure every future module will have the same `reverter`.

Reply:
Yeah I straight up forgot about it during coding.

Reply:
Thank you!